README.md

```diff
@@ -1 +1,177 @@
+
+"Contact form" section:
+
+![screenshot](/assets/images/contact_form_v3.png)
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The various stacks used for the project includes the following
+<details>
+  <summary>HTMl</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<details>
+<summary>Linters</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Learn how to translate a Figma design into a functional UI.**
+- **Utilize Flexbox to position various elements on a web page.**
+- **Utilize CSS Grid to arrange different elements on a web page.**
+- **Develop a portfolio website.**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+on MAC:
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+
+Example command:
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+- GitHub: [@yemidada](https://github.com/yemidada)
+- Twitter: [@naijavine](https://twitter.com/naijavine)
+- LinkedIn: [yemidada](https://linkedin.com/in/yemidada)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Add animation effect to the css (Animate.css)**
+- [ ] **Improve the designs**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can support us  by sharing our code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks you for checking out my repository
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **Do you need nodejs installed on your machine**
+
+  - Yes, nodejs need to be running properly on your machine
+
+- **Can you test linter on your local machine and github**
+
+  - Yes, you can do both
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+ This project is [MIT](./MIT.md) licensed. 
```

Comment:
- [OPTIONAL] Kindly resolve this url, it goes to the 404 page.

Reply:
Thank you very much. I have fixed the broken link @Olamarx